project.mml

```diff
@@ -2082,7 +2082,7 @@ Layer:
             "addr:housenumber" AS addr_housenumber,
             "addr:housename" AS addr_housename,
             tags->'addr:unit' AS addr_unit,
-            tags->'addr:flats' AS addr_flats,
+            CASE char_length(tags->'addr:flats') <= 9 WHEN TRUE THEN tags->'addr:flats' ELSE NULL END AS addr_flats,
```

Comment:
Don't use the switch-statement style case statement with an expression that evaluates to a boolean, instead do `CASE WHEN char_length(tags->'addr:flats') <= 9 THEN tags->'addr:flats' END AS addr_flats,`